src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -27,6 +34,34 @@
                  'edge': "_make_edge"
                 }
 
+class _ReusableDriver(webdriver.Remote):
+    def __init__(self, command_executor, sid, keep_alive=False, file_detector=None):
+
+        self.reuse_session_id = sid
+        super(_ReusableDriver, self).__init__(command_executor, desired_capabilities={},
+                                              browser_profile=None,
+                                              proxy=None, keep_alive=keep_alive,
+                                              file_detector=file_detector)
+
+    def stop_client(self):
+        if self.pid is not None and self.pid != '':
```

Comment:
Unfortunately not, because the selenium implementation is not consistent for different browsers.
Some of them use pid=None and some of them use pid=''.